src/os/README.md

```diff
@@ -112,7 +111,8 @@ inspiration for several decisions in this library.
 
 | C type | Hack type |
 |---|---|
-| Pointers to referential transparent C `struct`s | `vec` |
+| Pointers to referential transparent C `struct`s whose setters are unordered | `shape` |
+| Other pointers to referential transparent C `struct`s | `vec` |
```

Comment:
this wording is assuming that the majority of structs we care about:
- have setter functions as apposed to just containing properties that we can directly manipulate
- have order that matters beyond the C ABI - e.g. the order of properties in a `sockaddr_un` are ordered, this matters to the C ABI, but it doesn't matter for Hack at all.

First point I think needs revision; second is primarily a readability issue.

Reply:
Added "`struct`s of primitive types or other `struct`s"